pkgs/applications/video/ani-cli/default.nix

```diff
@@ -1,55 +1,45 @@
-{
-  fetchFromGitHub,
-  makeWrapper,
-  stdenvNoCC,
-  lib,
-  gnugrep,
-  gnused,
-  curl,
-  catt,
-  syncplay,
-  ffmpeg,
-  fzf,
-  aria2,
-  withMpv ? true,
-  mpv,
-  withVlc ? false,
-  vlc,
-  withIina ? false,
-  iina,
-  chromecastSupport ? false,
-  syncSupport ? false,
+{ fetchFromGitHub
+, makeWrapper
+, stdenvNoCC
+, lib
+, gnugrep
+, gnused
+, curl
+, catt
+, syncplay
+, ffmpeg
+, fzf
+, aria2
+, withMpv ? true, mpv
+, withVlc ? false, vlc
+, withIina ? false, iina
+, chromecastSupport ? false
+, syncSupport ? false
 }:
 
 assert withMpv || withVlc || withIina;
 
 stdenvNoCC.mkDerivation rec {
   pname = "ani-cli";
-  version = "4.8";
+  version = "4.9";
 
   src = fetchFromGitHub {
     owner = "pystardust";
     repo = "ani-cli";
     rev = "v${version}";
-    hash = "sha256-vntCiWaONndjU622c1BoCoASQxQf/i7yO0x+70OxzPU=";
+    hash = "sha256-7zuepWTtrFp9RW3zTSjPzyJ9e+09PdKgwcnV+DqPEUY=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
   runtimeDependencies =
-    let
-      player = [ ] ++ lib.optional withMpv mpv ++ lib.optional withVlc vlc ++ lib.optional withIina iina;
-    in
-    [
-      gnugrep
-      gnused
-      curl
-      fzf
-      ffmpeg
-      aria2
-    ]
-    ++ player
-    ++ lib.optional chromecastSupport catt
-    ++ lib.optional syncSupport syncplay;
+    let player = []
+        ++ lib.optional withMpv mpv
+        ++ lib.optional withVlc vlc
+        ++ lib.optional withIina iina;
+    in [ gnugrep gnused curl fzf ffmpeg aria2 ]
+      ++ player
+      ++ lib.optional chromecastSupport catt
+      ++ lib.optional syncSupport syncplay;
```

Comment:
If a formatter is used, make sure that you're using nixfmt-rfc-style

Reply:
Okay will do that in future! 👍️